docs/reference/modules/shard-ops.asciidoc

```diff
@@ -0,0 +1,69 @@
+[[shard-allocation-relocation-recovery]]
+=== Shard allocation, relocation, and recovery
+
+Each <<documents-indices,index>> in Elasticsearch is divided into one or more <<scalability,shards>>, each of which acts as a copy of the index and can be replicated across multiple <<modules-node,nodes>> to protect against hardware failures.
```

Comment:
`each of which acts as a copy of the index` this part is not correct when index has more than a single primary.
Does this imply replicas only? If so maybe we could mention this explicitly.